library/build.go

```diff
@@ -39,6 +39,7 @@ type Build struct {
 	Branch       *string `json:"branch,omitempty"`
 	Ref          *string `json:"ref,omitempty"`
 	BaseRef      *string `json:"base_ref,omitempty"`
+	HeadRef      *string `json:"head_ref,omitempty"`
```

Comment:
I think if we want to add a new field to this table, we'll also need to do so for the `database` type as well 👍 

Were you planning to add that in a separate PR?

You can find this other type in the `go-vela/types/database` package 😃 

https://github.com/go-vela/types/blob/6af7ad668990cadb758a68e5a9a532a1eb3237ca/database/build.go#L24-L53